lms/static/js/instructor_dashboard/util.js

```diff
@@ -379,7 +379,7 @@
                 url: listEndpoint,
                 success: function(data) {
                     if (data.tasks.length) {
-                        ths.createTaskListTable(ths.$table_running_tasks, data.tasks);
+                        createTaskListTable(ths.$table_running_tasks, data.tasks);
```

Comment:
It looked like so in coffeescript. https://github.com/edx/edx-platform/blob/7b9c566f3d4b489634a65e84c9cade12594b1219/lms/static/js/instructor_dashboard/util.js#L32

I'm afraid changing other rather than reverting back to it was would introduce more bugs. Very much 30% were not covered by tests.